LayoutTests/http/tests/dom/serialize-fragment-attributes-order.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+<body>
+<script src="/js-test-resources/js-test-pre.js"></script>
+<script>
+description("Verifies the order of attributes in markup is the same as the order they are added");
+
+if (window.testRunner) {
```

Comment:
All of this `if` scope should be replaced with:
```
jsTestIsAsync = true;
```

We should never use waitUntilDone() / notifyDone() with js-test.js.